### PR TITLE
CHAD kit  a human only traitor tiem

### PR DESCRIPTION
**Pull request by KazooBard**
## About The Pull Request

This item adds a human exclusive bundle including the following items, as well as introduces two new hard to get chemicals which are very potent toxins:

The cat bonker, a flyswatter for cats!
The mothswatter, self explainatory, deals flyswatter levels of damage (only to the species, same as the bonker)
A spray bottle of lube for the lack of utility in the kit
A spray bottle of human mutation toxin (stable mutation toxin)
A chemsprayer loaded with 999 units of clf3/moth-b-gon/cat-b-gon. The spray mode has extended range (max travel distance)
costs 14 TC. It is the CHAD, the catlike humanoid annihilation device. Untrue to it's name though...

... cat-b-gon affects all beings equally, as 14 TC for clf3 is not enough, and making something felinid exclusive has deemed itself to be a problem on the technical side. It deals heavy toxic and oxygen damage, has a low chance to induce sleepiness (5%). Should also cause mild effects mimicking those of the pepper spray, just worse.

Moth B gon is on the other hand moth race exclusive and causes effects similar to the pepper spray, just cut in half (stuns for 1 second making any hindrace allow your target to escape, the cat-b-gon is 1/4 original pepper spray values of jitteriness etc)

## Why It's Good For The Game

Both require the rare carpotoxin, which can be acquired from carps or emagged dispensers, but carps are not always there, and you could argue the carps can be made with xenobiology, but it is still a chemical and without a syringe gun and piercing syringes you can be countered by clothing, or even missing. 

Humans lack any sort of race specific item and bullying other races is their style. The kit is balanced with the toxins being spray based, so any security man with face coverance, or anyone with it in general has already dodged the strongest chemical in your arsenal. CLF3 is the only concern, but it is accessible widely.  

Also while making too many traitor items is bad, humans like a few other species are left with none exclusive to them, and what suits them better than bullying races and getting punished for it when someone places a gas mask on?

## Changelog
:cl:
add: moth-b-gon, cat-b-gon chems, along with recipes: 1 part carpotoxin, pestkiller and condensed capsaicin for mothbgon, 1 part carpotoxin, cocoa and 10 parts happiness for catbgon.
add: human specific traitor CHAD kit
add: cat bonker and mothswatter
/:cl:
KazooBard 
#Ssalty1179 on discord for details and stuff
mind the messy commits.

### CODE REVIEW

**Comment by bloons3** [comment]
So this kit is useless for completing any actual objectives and is just meant to harass other players?

**Comment by Effemu** [comment]
very good pr

> 
> 
> So this kit is useless for completing any actual objectives and is just meant to harass other players?

you take this kit if your targets are moth or felinid players

**Comment by KazooBard** [comment]
> So this kit is useless for completing any actual objectives and is just meant to harass other players?

First of all, why harrasing? 

Second, if your target is a moth or felinid it gives you a lot of advantage, but it is still a solid bet for regular objectives having a huge capacity. A chemist can buy this just to make a massive spray bottle container with good range and fill it with whatever.

Third, is it useless? Not completely, but I recall moths can buy a lamp. Now using this entire bundle can help you in a pinch if security forces have mothpeople in them, and if I get a little help, felinid security players are also evened out against you. Now is that a bad thing? Not at all, since it promotes gathering knowledge and creating weaknesses against players, while having counters such as the ranged spray being mostly nullified by clothes, and the close range swatters are swatters. They are not effective against anything else and provide big advantages when picking your fights. Now for a 14 TC item it is not bad, if you catch someone offguard or use the chems inside as poison by extracting them you can do much more.

Not to brag, but "know your enemy" is a big part of winning, isn't it?

**Comment by Naevii** [comment]
I am sorry, but this is absolutely terrible. 
It encourages toxicity rather than creating actually fun situations. (even the name points that toxin behavior towards other races players is cool, while it is not)

The way i see and understand this, this kit will be only fun for the person purchasing it.

**Comment by SofterStone** [comment]
> I am sorry, but this is absolutely terrible.
> It encourages toxicity rather than creating actually fun situations. (even the name points that toxin behavior towards other races players is cool, while it is not)
> 
> The way i see and understand this, this kit will be only fun for the person purchasing it.

yeeeaaah getting killed with desword is totally fun for both

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 1]
```diff
@@ -151,3 +151,18 @@
...
+		H.adjustToxLoss(8)
+		H.adjustOxyLoss(8)
+		if(prob(10))
+			H.drowsyness += 1
+		H.visible_message("<span class='danger'>You have to get this chocolate out of your system!!!</span>")
+
+
+/datum/species/human/felinid/check_species_weakness(obj/item/weapon, mob/living/attacker)
+	if(istype(weapon, /obj/item/melee/flyswatter/cat))
+		return 29 //flyswatters deal 20x damage to moths
```
Comment (Archanial) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:167]: ???

File: code/modules/mob/living/carbon/human/species_types/felinid.dm [hunk 2]
```diff
@@ -151,3 +151,18 @@
 
 	if(!silent)
 		to_chat(H, "You are no longer a cat.")
+
+/datum/species/human/felinid/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	. = ..()
+	if(chem.type == /datum/reagent/toxin/catbgon)
```
Comment (Archanial) [inline code/modules/mob/living/carbon/human/species_types/felinid.dm:157]: this should be in chemical on metabolize

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 3]
```diff
@@ -36,7 +36,34 @@
...
+
+
 /datum/species/moth/check_species_weakness(obj/item/weapon, mob/living/attacker)
 	if(istype(weapon, /obj/item/melee/flyswatter))
 		return 9 //flyswatters deal 10x damage to moths
 	return 0
+
+/datum/species/moth/check_species_weakness(obj/item/weapon, mob/living/attacker)
+	if(istype(weapon, /obj/item/melee/flyswatter/moth))
+		return 29 //flyswatters deal 30x damage to moths
```
Comment (Archanial) [inline code/modules/mob/living/carbon/human/species_types/mothmen.dm:68]: same here, what does 29 mean

**Review by Archanial — CHANGES_REQUESTED** [review]
terrible idea

**Comment by KazooBard** [comment]
> I am sorry, but this is absolutely terrible. 
> 
> It encourages toxicity rather than creating actually fun situations. (even the name points that toxin behavior towards other races players is cool, while it is not)
> 
> 
> 
> The way i see and understand this, this kit will be only fun for the person purchasing it. 
> 
> 

Idea on how to improve it? Also a lot of items are fun for the person purchasing it only. I do not remember having fun being shot.

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm (hunk 3 above)
Comment (KazooBard) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:68]: Bonus damage

**Comment by Sinestia** [comment]
I dunno what this adds that's worth bloating the traitor item list even more but eh

Also like, every string has a bunch of typoes. 

You messed up your CL too, name goes to the right side of the first :cl:

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm (hunk 3 above)
Comment (Archanial) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm:68]: this should be in mothswatter not in species

**Comment by Archanial** [comment]
Also this touches species code. Unathorized changes get closed.

**Comment by francinum** [comment]
Remember when I said in a thread that species are generally just used as an excuse to be assholes to each other?

This is what I meant.

**Comment by KazooBard** [comment]
Ok

**Pull request closed without merging**